eZ/Publish/Core/FieldType/Image/ImageStorage.php

```diff
@@ -122,6 +122,8 @@ public function storeFieldData(VersionInfo $versionInfo, Field $field, array $co
             $field->value->externalData = null;
         }
 
+        if(!isset($field->value->data['id'])){return false;}
```

Comment:
Checking it here is a bit late, I believe that bug is somewhere [here](https://github.com/ezsystems/ezpublish-kernel/blob/6.7/eZ/Publish/Core/FieldType/Image/ImageStorage.php#L109) either because `if` is not handling lack of `data['id']` or even the fact that `$this->data` is not null when image is empty (I think the latter is the one)